reports/Google-2018-report.md

```diff
@@ -13,13 +15,18 @@ prioritize transparency, diversity, and reproducibility to foster collaboration.
 Imagine you are a carrot farmer trying to make dinner. You could make carrot soup by yourself, but you could also share ingredients 
```

Comment:
I don't think this analogy is necessary, but happy to keep if you feel it enhances the message !

Reply:
Yeah may have been a bit of overkill! but just added it to make the concept more tangible